code/src/vs/workbench/contrib/webview/browser/pre/index-no-csp.html

```diff
@@ -333,6 +333,12 @@
 
 				const hostname = location.hostname;
 
+				// Run if we are on the same host.
+				const parent = new URL(parentOrigin);
+				if (parent.hostname === hostname) {
+					return start(parentOrigin);
+				}
+
```

Comment:
I think it makes sense to provide a logic for automatic rebase
@azatsarynnyy @vitaliy-guliy wdyt?

Reply:
Considering that [it's changing about every 2 weeks](https://github.com/microsoft/vscode/commits/main/src/vs/workbench/contrib/webview/browser/pre/index.html), it would be nice [to add a rebase rule](https://github.com/che-incubator/che-code/blob/main/rebase.sh#L219) for resolving the potential merge conflicts automatically, so fewer distractions for us.

Reply:
Done